coderd/coderdtest/coderdtest.go

```diff
@@ -176,6 +177,14 @@ func NewOptions(t *testing.T, options *Options) (func(http.Handler), context.Can
 	if options.Database == nil {
 		options.Database, options.Pubsub = dbtestutil.NewDB(t)
 	}
+	// TODO: remove this once we're ready to enable authz querier by default.
+	if strings.Contains(os.Getenv("CODER_EXPERIMENTS_TEST"), "authz_querier") {
+		panic("Coming soon!")
+		if options.Authorizer != nil {
+			options.Authorizer = &RecordingAuthorizer{}
```

Comment:
The original authorizer is needed for some routes like app execution, but we can figure that out in authzquerier branch.